packages/flutter_tools/lib/src/service_protocol.dart

```diff
@@ -8,9 +8,13 @@ import 'device.dart';
 
 /// Discover service protocol ports on devices.
 class ServiceProtocolDiscovery {
-  /// [logReader] A [DeviceLogReader] to look for Observatory messages in.
-  ServiceProtocolDiscovery(DeviceLogReader logReader)
-      : _logReader = logReader {
+  static const String kObservatoryService = 'Observatory';
+  static const String kDiagnosticService = 'Diagnostic server';
```

Comment:
Will this print out every time the user runs an app? I'm not sure it's going to be immediately clear to them what the diagnostic server is or does.